src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/CertificateAccess/Ressources/CertificateSchema.json

```diff
@@ -38,7 +38,7 @@
     },
     "v": {
       "description": "Vaccination Group",
-      "type": ["array", null],
+      "type": ["array", "null"],
```

Comment:
Does that work now?
I needed to change "null" to null to make the schema validation work.

Reply:
I only tested with the certificate generated from here as per the tech spec, **cwa tc gen --preset 25**
the validation works but the certificate has an invalid signature, so it should work, i will retest it